backend/src/tests/utils/user.py

```diff
@@ -17,9 +17,9 @@ async def create_test_user(db: AsyncSession) -> User:
             full_name=TEST_USER_NAME,
             email=TEST_USER_EMAIL,
             password=TEST_USER_PASSWORD,
+            is_admin=False

```

Comment:
is_admin should be passed into the crud.user.create function and that function should be updated to take a is_admin bool parameter, do not pass it in to the UserCreate model